torch/_torch_docs.py

```diff
@@ -6819,10 +6875,12 @@ def merge_dicts(*dicts):
            r"""
 sparse_coo_tensor(indices, values, size=None, *, dtype=None, device=None, requires_grad=False) -> Tensor
 
-Constructs a sparse tensors in COO(rdinate) format with non-zero elements at the given :attr:`indices`
-with the given :attr:`values`. A sparse tensor can be `uncoalesced`, in that case, there are duplicate
-coordinates in the indices, and the value at that index is the sum of all duplicate value entries:
-`torch.sparse`_.
+Constructs a :ref:`sparse tensors in COO(rdinate) format
+<sparse-coo-docs>` with specified elements at the given
+:attr:`indices` with the given :attr:`values`. A sparse tensor can be
```

Comment:
Does this note about sparse tensors being coalesced or not really belong here? Seems more appropriate for a note on sparse tensors?

Reply:
`sparse_coo_tensor` always returns a tensor that is not coalesced.

Replaced the sentence with a note:
```
This function returns an :ref:`uncoalesced tensor <_sparse-uncoalesed-coo-docs>`.
```